### DISCUSSION TITLE
Simulating a Thread network with OpenThread on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by MartinsMihaeljans** [openthread/openthread#8960] [Q&A] [answered]
Hello, I tried to go trough codelabs - https://openthread.io/codelabs/openthread-simulation-posix#5
I stopped at section **Manage the network with OpenThread Daemon** at step **2. Start ot-daemon** because of an error.
I understand that the codelab was tested on Ubuntu 14, but my VM is ubuntu 22.04, therefore, I'm guessing the error might be brought up by some version mismatch. Could someone please look this up and explain what could be the problem?
Here is the command and output:

`administrator@server:~/src/openthread$ sudo ./build/posix/src/posix/ot-daemon -v 'spinel+hdlc+forkpty:///build/simulation/examples/apps/ncp/ot-rcp?forkpty-arg=2'`

./build/posix/src/posix/ot-daemon[12776]: Running OPENTHREAD/thread-reference-20200818-2421-g0ca01ae50; POSIX; Apr 13 2023 11:57:50
./build/posix/src/posix/ot-daemon[12776]: Thread version: 4
./build/posix/src/posix/ot-daemon[12776]: 49d.19:58:03.280 [C] Platform------: Init() at hdlc_interface.cpp:150: InvalidArgument

**Comment by wgtdkp** [comment, marked as answer]
The path should be `'spinel+hdlc+forkpty://build/simulation/examples/apps/ncp/ot-rcp?forkpty-arg=2'` - it starts with **//** but not `///`

**Reply by MartinsMihaeljans** [reply]
Thank you very much, this sure was the problem as the command gave similar output as in the example.

**Reply by jwhui** [reply]
I submitted an update to the docs: https://github.com/openthread/ot-docs/pull/116
